core/templates/pages/learner-dashboard-page/suggestion-modal/learner-dashboard-suggestion-modal.component.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for learner dashboard suggestion modal.
+ */
+
+import { downgradeComponent } from '@angular/upgrade/static';
+import { Component, Input } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'oppia-learner-dashboard-suggestion-modal',
+  templateUrl: './learner-dashboard-suggestion-modal.component.html',
+  styleUrls: []
+})
+export class LearnerDashboardSuggestionModalComponent {
```

Comment:
maybe use ConfirmOrCancelModal for this one

Reply:
done!